i18n/en/docusaurus-plugin-content-docs/current/guides/setup.mdx

```diff
@@ -0,0 +1,215 @@
+
+**If you do start making changes, remember the golden rule:**
+
+:::danger
+**Since parameters sometimes influence each other, it is STRONGLY encouraged to only make changes to one setting at at time, and test the effects of each change before making further modifications.**
+:::
+::::
+
+### Syntax Error in evcc.yaml
+
+Yaml is very sensitive to syntax - and errors don't always catch the eye straight away. Linters such as (https://onlineyamltools.com/validate-yaml) can be super useful, and are worth checking to find simple mistakes.
+
+## Common Problems
+
+### Something's not working. What now?
+
+We have a [Community Support](https://github.com/evcc-io/evcc/discussions) forum, with developers and users standing by to help solve common issues.
+
+When making a post, it would be super useful to give as much information as possible on the following:
+
+- As precise an explanation as possible as to what your problem is
+
+- Which devices (vehicles, meters, chargers) are in use?
+
+- Your full, sanitised `evcc.yaml`
+
+<details>
+<summary>
+   Helper comamnd (*please make sure all usernames, passwords, and other secrets are removed!*).
+</summary>
+
+`egrep -v '^#|^\s+#' evcc.yaml |grep . |sed -E 's/(password|user|token|url|host|broker).*/\1: *****/'`
+
+</details>
+
+  - Alternatively, you can use `evcc dump --cfg >~/config.txt` to create an anonymized configuration file at `~/config.txt`.
+
+
+**⛑️ First Aid**
+
+We've added a useful helper utility to evcc to help with the above. By entering `evcc discuss` in your terminal, our helper wizard will help you create a new discussion topic in our "First Aid" section.
+This discussion will automatically include any relevant error messages, and the anonymized configuration file - you only need to supply a meaningful title and a description of your problem.
+
+This First Aid utility can only be use for the initial creation of a discussion thread. If you need to supply further information to a topic already in progress, you'll need to do this manually.
+
+### How do I create a log fire for error analysis?
+
+In its standard configuration, evcc only issues log messages if an _error_ occurs (see also the [`log`](docs/reference/configuration/log#log) configuration key).
+
+When evcc is started directly from the console, log messages are printed directly to the screen (`STDOUT`).
+
+You can test out different parts of your installation using `evcc charger`, `evcc meter`, and `evcc vehicle`. Note that the log level defined in your `evcc.yaml` is used with these commands, but you can override it if necessary:
+
+`Example: evcc -l debug meter`
+
+If evcc is started as a Linux system service (as per [Linux Autostart](/docs/installation/manual#autostart)), you should be able to find your logs using the following commands:
+
+- Follow the log in real time
+  `sudo journalctl -fau evcc`
```

Comment:
Line wraps don't work. Not quite sure why.

![Bildschirmfoto 2023-08-22 um 15 34 56](https://github.com/evcc-io/docs/assets/152287/47894446-1337-4b5b-b643-7607dac83042)

Reply:
Probably worth investigating why this is happening with the element, might need to set a wrapping CSS rule on inline code elements

For now, resolved in [988f08e](https://github.com/evcc-io/docs/pull/426/commits/988f08e72acab3818a800f013e87c923614d40d5)